decidim-participatory_processes/lib/decidim/participatory_processes/engine.rb

```diff
@@ -261,26 +261,36 @@ class Engine < ::Rails::Engine
           content_block.cell = "decidim/proposals/content_blocks/highlighted_proposals"
```

Comment:
While reviewing #10573, i realised that we could add a check to see if the proposals module is defined.